pgvectorscale/src/util/tape.rs

```diff
@@ -121,19 +121,6 @@ mod tests {
                 node_page
             };
 
-            {
-                let mut tape = Tape::resume(&indexrel, PageType::SbqMeans);
-                let ip = tape.write(&[99]);
-                assert_eq!(
-                    ip.block_number, tape.current,
-                    "Tape block number should match IP"
-                );
-                assert_ne!(
-                    tape.current, node_page,
-                    "Data can only be written to page of its type"
-                );
-            }
-
```

Comment:
It would be better to update `SbqMeans` to `SbqMeansV1` here instead of deleting the whole test.